front/playwright.config.ts

```diff
@@ -14,6 +14,7 @@ export default defineConfig({
   /* Maximum time one test can run for. */
   timeout: process.env.CI ? 90 * 1000 : 180 * 1000, // 90 seconds in CI, otherwise 180 seconds
   expect: {
+    toHaveScreenshot: { maxDiffPixels: 20 },
```

Comment:
There’s a small pixel difference between running tests locally and in CI, usually between 1 and 18 pixels. I added a 2-pixel buffer to avoid flaky failures, so the tolerance is 20.

Reply:
It's true that we have the same web browsers versions but we can have different OS enviremeent also the CI lack hardware acceleration which make graphic rendering a little bit differente 

Actual image from the CI: 
![all-settings-GEV-actual](https://github.com/user-attachments/assets/ec3c30c9-7efd-41be-a7a1-0a513052d054)
Expected image:
![all-settings-GEV-expected](https://github.com/user-attachments/assets/83fbc8ac-149f-4ac7-b77c-cb3c8e4ee7cc)
The difference between both images ( 18 pixels differences displayed in red ):
![all-settings-GEV-diff](https://github.com/user-attachments/assets/3f85f4df-2dd4-477b-a470-b0d1695e4946)

Reply:
can’t you create the expected images from a similar environment ? so that the CI actually tests what we should get taking the environment into account